index.html

```diff
@@ -9,7 +9,8 @@
     <link rel="stylesheet" href="header.css">
     <link rel="stylesheet" href="footer.css">
     <link rel="stylesheet" href="btn.css">
-
+    <link rel="stylesheet" href="menu.css">
+    <link rel="stylesheet" href="cookie.css">
```

Comment:
A lot of CSS files linked. Consider merging them to one single CSS file, to keep HTTP requests down. Webpack or Sass does the trick :)